core/java/com/google/census/Census.java

```diff
@@ -41,7 +41,12 @@ public static CensusContext getCurrent() {
     return CONTEXT_FACTORY.getCurrent();
   }
 
-  /** Creates a {@link CensusContext} from the given on-the-wire encoded representation. */
+  /**
+   * Creates a {@link CensusContext} from the given on-the-wire encoded representation.
+   *
+   * <p>Should be the inverse of {@link CensusContext#serialize()}. The serialized representation
```

Comment:
The Google style guide specifies that `<p>` should be on the same line as the new paragraph: https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs